config/routes.rb

```diff
@@ -24,5 +24,8 @@
     get 'random', on: :collection
   end
 
-  resources :groups
+  resources :groups do
+    resources :group_members
+    resources :invites, only: %i[create destroy]
```

Comment:
`destroy` is going to be implemented later? I'd keep it away until than (and maybe put TODO instead)